ownCloud/Client/Actions/Actions+Extensions/CollaborateAction.swift

```diff
@@ -24,13 +24,29 @@ class CollaborateAction: Action {
 	override class var identifier : OCExtensionIdentifier? { return OCExtensionIdentifier("com.owncloud.action.collaborate") }
 	override class var category : ActionCategory? { return .normal }
 	override class var name : String { return OCLocalizedString("Sharing", nil) }
-	override class var locations : [OCExtensionLocationIdentifier]? { return [.keyboardShortcut, .contextMenuSharingItem, .moreItem, .moreDetailItem, .accessibilityCustomAction] }
+	override class var locations : [OCExtensionLocationIdentifier]? { return [.keyboardShortcut, .contextMenuSharingItem, .moreItem, .moreFolder, .moreDetailItem, .accessibilityCustomAction] }
```

Comment:
I think the change in this line should be the only change necessary to implement the change. The rest should not be needed.